go.mod

```diff
@@ -11,32 +11,162 @@ require (
 	github.com/docker/go-metrics v0.0.1 // indirect
 	github.com/docker/libtrust v0.0.0-20160708172513-aabc10ec26b7 // indirect
 	github.com/fluxcd/helm-controller/api v0.14.1
-	github.com/fluxcd/pkg/apis/kustomize v0.3.0
-	github.com/fluxcd/pkg/apis/meta v0.10.1
-	github.com/fluxcd/pkg/runtime v0.12.2
-	github.com/fluxcd/source-controller/api v0.19.2
+	github.com/fluxcd/pkg/apis/kustomize v0.3.1
+	github.com/fluxcd/pkg/apis/meta v0.10.2
+	github.com/fluxcd/pkg/runtime v0.12.3
+	github.com/fluxcd/source-controller/api v0.20.0
 	github.com/garyburd/redigo v1.6.3 // indirect
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v1.2.2
 	github.com/gofrs/uuid v4.1.0+incompatible // indirect
-	github.com/gorilla/handlers v1.5.1 // indirect
 	github.com/hashicorp/go-retryablehttp v0.6.8
 	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0 // indirect
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.15.0
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.17.0
 	github.com/spf13/pflag v1.0.5
 	github.com/yvasiyarov/go-metrics v0.0.0-20150112132944-c25f46c4b940 // indirect
 	github.com/yvasiyarov/gorelic v0.0.7 // indirect
 	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20160601141957-9c099fbc30e9 // indirect
-	helm.sh/helm/v3 v3.7.1
-	k8s.io/api v0.22.2
-	k8s.io/apiextensions-apiserver v0.22.2
-	k8s.io/apimachinery v0.22.2
-	k8s.io/cli-runtime v0.22.2
-	k8s.io/client-go v0.22.2
-	rsc.io/letsencrypt v0.0.3 // indirect
-	sigs.k8s.io/controller-runtime v0.10.2
+	helm.sh/helm/v3 v3.7.2
```

Comment:
We can't bump Helm in this PR, please see https://github.com/fluxcd/helm-controller/pull/380

Reply:
Yep, sorry, I forgot about that.